packages/cli/src/authentication/index.ts

```diff
@@ -59,6 +59,13 @@ export const prompts = (ctx: AuthenticationGeneratorArguments) => [
     type: 'input',
     when: !ctx.service,
     message: 'What is your authentication service name?',
+    default: 'user'
```

Comment:
The service name is mostly for variable naming. So it'll be `UserData` `User`, `userResolver` etc. instead of the plural `UsersData`, `usersResolver`.